Sources/PackagePlugin/PublicAPI/TargetBuildContext.swift

```diff
@@ -69,15 +69,32 @@ public final class TargetBuildContext: Decodable {
     /// target on which the package plugin target depends. Returns nil, but
     /// does not throw an error, if the tool isn't found. Plugins that re-
     /// quire the tool should emit an error diagnostic if it cannot be found.
-    public func lookupTool(named name: String) throws -> Path {
-        // TODO: Rather than just appending the name, this should instead use
-        // a mapping of tool names to paths (passed in from the context).
-        return self.toolsDir.appending(name)
+    public func tool(named name: String, line: UInt = #line) throws -> Tool {
+        if let tool = self.tools[name] { return tool }
```

Comment:
case sensitive okay?

Reply:
We should discuss this.  I am pretty sure that the proposal said that it's case sensitive, but will double-check.

Reply:
The proposal doesn't say, but I think we should clarify it to be case sensitive.  Target and product names are case sensitive, so I think that would make sense.